service/src/main/java/uk/gov/hmcts/reform/fpl/utils/PeopleInCaseHelper.java

```diff
@@ -32,4 +36,18 @@ public static String getFirstRespondentLastName(List<Element<Respondent>> respon
                 representative.getAddress().getAddressAsString(", ")))
             .collect(toList());
     }
+
+    public static String getAllocatedJudgeTitle(CaseData data) {
+        if (data.getAllocatedJudge().getJudgeTitle() == OTHER) {
+            return data.getAllocatedJudge().getOtherTitle();
+        }
+        return data.getAllocatedJudge().getJudgeTitle().getLabel();
+    }
+
+    public static String getAllocatedJudgeName(CaseData data) {
+        if (data.getAllocatedJudge().getJudgeTitle() == MAGISTRATES) {
+            return data.getAllocatedJudge().getJudgeFullName();
+        }
+        return data.getAllocatedJudge().getJudgeLastName();
+    }
```

Comment:
Do you think these methods would be better on the `Judge` object itself?

Reply:
Yeah that makes sense to me. :)